### PR TITLE
add variable to enable oms_agent

### PR DESCRIPTION
**Pull request by sergiomcalzada**
Add a variable to enable the Azure Monitor

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -268,3 +268,9 @@ variable vnet_subnets {
     }
   ]
 }
+
+variable oms_log_analytics_workspace_id {
+  type        = string
+  description = "(Required) The Log Analytics Workspace id when the OMS should store logs."
```
Comment (sesispla) [inline variables.tf]: Please change it to "(Optional)"

File: variables.tf [hunk 2]
```diff
@@ -268,3 +268,9 @@ variable vnet_subnets {
...
+  default     = false
```
Comment (sesispla) [inline variables.tf]: Please change the default value to "null" (without quotes)

**Review by sesispla — CHANGES_REQUESTED** [review]
Please adjust the description and default values. Thanks!!

File: variables.tf (hunk 1 above)
Comment (sergiomcalzada) [reply variables.tf]: done

File: variables.tf (hunk 2 above)
Comment (sergiomcalzada) [reply variables.tf]: done

**Review by sesispla — APPROVED** [review]
Perfect, thank you!!

**Pull request merged**